staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -321,10 +352,17 @@ type DeleteOptions struct {
 	// +optional
 	Preconditions *Preconditions `json:"preconditions,omitempty" protobuf:"bytes,2,opt,name=preconditions"`
 
+	// Deprecated: please use the PropagationPolicy.
```

Comment:
Are you sure that we want to deprecate this right away?
Shouldnt we let PropagationPolicy be there for a release, stabilize it and then deprecate OrphanDependents?